platforms/jvm/plugins-application/src/main/java/org/gradle/api/internal/plugins/StartScriptTemplateBindingFactory.java

```diff
@@ -157,7 +158,11 @@ private String escapeWindowsJvmOpt(String jvmOpts) {
     private enum ScriptBindingParameter {
         APP_NAME("applicationName"),
         OPTS_ENV_VAR("optsEnvironmentVar"),
+
+        @SuppressWarnings("DeprecatedIsStillUsed")
+        @Deprecated
         EXIT_ENV_VAR("exitEnvironmentVar"),
```

Comment:
Why are we leaving this around? I don't see why we need to continue adding a templating binding if we removed the variable from the template

Reply:
Removed.

Reply:
Looking back at this, I realized it was possible for users to configure their own templates -- and they still may expect this variable to be present. 

Sorry for the back-and-forth here, but we might need to keep this around.

Reply:
And it's back.